### ISSUE TITLE
Crush App.

### ISSUE THREAD

**Issue by 345ml** [rmrs/react-native-firebaseui#9]
Rendering ImageView or PhotoView will crash the application.
Is there sample code for implementation?

**Comment by erezrokah** [comment]
I'll try to add an example app soon. 
Can you supply more information in the mean time?
1. Are you on Android or iOS?
2. Example of the code you're using.
3. The crash error you're getting.
4. Any other log you might think of

**Comment by 345ml** [comment]
Sorry for lack of information.
I tried it on Android.

### Package.json.
```
"react": "16.3.1",
"react-native": "0.55.4",
"react-native-firebase": "^5.1.0",
"react-native-firebaseui": "^0.0.1-alpha7",
```

### My code.
In the first place, usage may be wrong...
```
import React, { Component } from 'react';
import firebase from 'react-native-firebase';
import { ImageView, PhotoView } from 'react-native-firebaseui';

const Index = (props) => {
  const imageProps = this.props;

  return (
    <PhotoView
      {...imageProps}
      path="firebase/storage/path"
      timestamp={0}
      resizeMode="cover"
    />
  );
};

export default Index;
```

### Error
```
Error while updating property 'timestamp' of a view managed by: RCTFirebasePhotoView

method io.rumors.reactnativefirebaseui.storage.FirebasePhotoViewManager.setTimestamp argument 1 has type io.rumors.reactnativefirebaseui.storage.ExtendedImageView, got io.rumors.reactnativefirebaseui.storage.ExtendedPhotoView
updateViewProp
    ViewManagersPropertyCache.java:92
setProperty
    ViewManagerPropertyUpdater.java:129
updateProps
    ViewManagerPropertyUpdater.java:48
updateProperties
    ViewManager.java:32
createView
    NativeViewHierarchyManager.java:232
execute
    UIViewOperationQueue.java:152
dispatchPendingNonBatchedOperations
    UIViewOperationQueue.java:1012
doFrameGuarded
    UIViewOperationQueue.java:983
doFrame
    GuardedFrameCallback.java:29
doFrame
    ReactChoreographer.java:134
doFrame
    ChoreographerCompat.java:105
run
    Choreographer.java:947
doCallbacks
    Choreographer.java:761
doFrame
    Choreographer.java:693
run
    Choreographer.java:935
handleCallback
    Handler.java:873
dispatchMessage
    Handler.java:99
loop
    Looper.java:193
main
    ActivityThread.java:6669
invoke
    Method.java
run
    RuntimeInit.java:493
main
    ZygoteInit.java:858
```


Try delete `timestamp={0}` and the application crashed.

**Comment by erezrokah** [comment]
Hey @345ml, you were right there was a casting bug in the Android code.
Should be fixed by #10 
I've also added an example code.
Please re-open if you have anymore issues

**Comment by 345ml** [comment]
@erezrokah 
Thenk you!
Great!!
